src/Compilers/Core/Portable/Diagnostic/LocalizableResourceString.FixedLocalizableString.cs

```diff
@@ -15,7 +15,7 @@ private sealed class FixedLocalizableString : LocalizableString
 
             private readonly string _fixedString;
 
-            public static FixedLocalizableString Create(string fixedResource)
+            public static FixedLocalizableString CreateFixed(string fixedResource)
```

Comment:
>CreateFixed [](start = 49, length = 11)

I think it would be better to not add ```internal static LocalizableString Create(string fixedResource)``` in ```LocalizableString``` instead of renaming the method.

Reply:
Sounds reasonable - I have removed that API and instead added an explicit cast at the callsite.